src/gui/redux/model/model.spec.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-env jest */
+import { SET_MODEL, SET_VALUE, setModel, setValue } from './model.actions'
+import reducer from './model'
+
+describe('gui/redux/model', () => {
+  describe('actions', () => {
```

Comment:
Actions are used into reducer tests, to avoid testing twice the actions : 
 - Either you do not test it alone
 - Or you do not use them into reducer tests